webpack.config.js

```diff
@@ -0,0 +1,34 @@
+const path = require("path");
+
+// VSCode does not support this encodings, so we don't need to bundle them.
+// This saves us some bytes, which is important for vscode-web.
+const removeUnsupportedEncodings = [
+  "utf7.js",
+  "utf32.js",
+  "tables/cp949.json",
+].map((file) => {
+  return {
+    test: path.resolve(__dirname, "node_modules/iconv-lite/encodings", file),
+    use: "fake-loader",
+  };
+});
+
+module.exports = {
+  entry: "iconv-lite",
+  devtool: "inline-source-map",
```

Comment:
This comes from `semver-umd` module and seem to increase the file output size greatly. Do you know if it is on purpose or just an oversight?